src/__tests__/components/CountriesContainer.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import renderer from 'react-test-renderer';
+import store from '../../__mocks__/configureStore';
+import CountriesContainer from '../../components/CountriesContainer';
+
+describe('Countries snapshot', () => {
+  test('Check if renders correctly', () => {
+    const tree = renderer
+      .create(
+        <Provider store={store}>
+          <BrowserRouter>
+            <CountriesContainer />
```

Comment:
Please use `grill sans` font for the body and title on both pages. Kindly use the same typography as in the design for all the pages.

Reply:
Ok, changed that.